dask/bag/text.py

```diff
@@ -0,0 +1,102 @@
+import io
+import os
+
+from toolz import concat
+
+from ..utils import infer_compression, system_encoding
+from ..delayed import delayed
+from ..bytes.compression import files as cfiles, seekable_files
+from ..bytes import open_text_files, read_bytes
+from .core import from_delayed
+
+delayed = delayed(pure=True)
+
+
+def read_text(path, blocksize=None, compression='infer',
+              encoding=system_encoding, errors='strict',
+              linedelimiter=os.linesep, collection=True, **kwargs):
```

Comment:
That's a good point.  Maybe we keep this as `None` and let the `bytes` backends use their own defaults.